pkg/client/clientset/versioned/scheme/register.go

```diff
@@ -48,5 +48,8 @@ func init() {
 // After this, RawExtensions in Kubernetes types will serialize kube-aggregator types
 // correctly.
 func AddToScheme(scheme *runtime.Scheme) {
-	scalingv1alpha1.AddToScheme(scheme)
+	err := scalingv1alpha1.AddToScheme(scheme)
```

Comment:
would it be preferable to panic during init if returning `err`?

Reply:
I think so but I think I'll revert some of these changes to make the linter happy, and move those into a separate pr so i can yeet things upstream